src/emphasis/alert/AlertItem.tsx

```diff
@@ -8,58 +8,50 @@
  *     http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Add Storybook usecase to test his scenario.

Reply:
Added storybook usecase for this scenario.